Makefile

```diff
@@ -329,3 +329,12 @@ e2e-test:
 .PHONY: e2e-test-v2
 e2e-test-v2:
 	BUILD_V2=1 make e2e-test
+
+.PHONY: pod-failover-test-containers
+pod-failover-test-containers:
+	go build -a -mod vendor -o _output/${ARCH}/workloadPod ./test/podFailover/workload 
```

Comment:
Building with `CGO_ENABLED=0` will create an executable with no dependencies on the C-runtime libraries. This will allow you to use a smaller base image which will reduce initial time to schedule the pod because it will take less time to download the image. The ubuntu base image is relative large. Since neither the controller nor the pod depend on any other executables, you might be able to get away with `gcr.io/distroless/static:latest` for the base image.